lib/tooling/pahole-tool.js

```diff
@@ -25,15 +25,19 @@
 
 const
     utils = require('../utils'),
+    fs = require('fs-extra'),
+    denodeify = require('denodeify'),
     BaseTool = require('./base-tool');
 
 class PaholeTool extends BaseTool {
     constructor(toolInfo, env) {
         super(toolInfo, env);
+
+        this.stat = denodeify(fs.stat);
```

Comment:
As discussed with @apmorton on Slack, there should be no need to senodeify the functions coming from `fs-extra`

Reply:
merged stuff from a while back, will fiddle around and eliminate